### PR TITLE
Add Tox21 consist of 12 assay with URI include

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

**Comment by MicPie** [comment]
@phalem I just added the changes black needs to pass the commit hooks. You can simply get black to add the changes if you run `black .` in the main directory (so no need to search for the changes in the logs on GitHub).

**Comment by MicPie** [comment]
Note: We need to define the target description properly. So far I changed it to include the assay name. Maybe we should discuss a more unspecific setup?

**Comment by phalem** [comment]
@MicPie  Great work! You are amazing..!

File: data/nr_er_lbd_tox21/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      names:
+          - noun: NR-ER-LBD toxicity
+          - noun: NR-estrogen receptor alpha ligand binding domain toxicity
+          - noun: estrogen receptor alpha ligand binding domain toxicity
+          - verb: activity in the NR-ER-LBD toxicity assay
+          - verb: activity in the NR-estrogen receptor alpha ligand binding domain assay
+          - verb: activity in the estrogen receptor alpha ligand binding domain assay
+          - adjective: toxic in the NR-ER-LBD assay
+          - adjective: toxic in the NR-estrogen receptor alpha ligand binding domain assay
+          - adjective: toxic in the estrogen receptor alpha ligand binding domain assay
```
Comment (kjappelbaum) [inline data/nr_er_lbd_tox21/meta.yaml:21]: do we add `is toxic`?

Comment (kjappelbaum) [reply data/nr_er_lbd_tox21/meta.yaml:21]: same comments for the other files

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
i have one question about how we should use `verb` - do we include `is` or not? Otherwise, lgtm!

File: data/nr_ahr_tox21/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    response pathways.
+targets:
+    - id: toxicity_NR-AhR
+      description: whether it shows activity in a aryl hydrocarbon receptor toxicity assay (1) or not (0)
+      units:
+      type: boolean
+      names:
+          - noun: NR-AhR toxicity
+          - noun: aryl hydrocarbon receptor toxicity
+          - verb: activity in the aryl hydrocarbon receptor toxicity assay
```
Comment (kjappelbaum) [inline data/nr_ahr_tox21/meta.yaml:15]:
```suggestion
          - verb: is active  in the aryl hydrocarbon receptor toxicity assay
```

**Comment by kjappelbaum** [comment]
@MicPie I'd fix the remaining validation complains in a separate PR

File: data/nr_ar_tox21/transform.py [hunk 3]
```diff
@@ -57,7 +57,7 @@ def get_and_transform_data():
                 "names": [  # names for the property (to sample from for building the prompts)
                     {"noun": "NR-AR toxicity"},
                     {"noun": "NR-androgen receptor toxicity"},
-                    {"verb": "toxic in a androgen receptor toxicity assay"},
+                    {"verb": "is toxic in a androgen receptor toxicity assay"},
```
Comment (MicPie) [inline data/nr_ar_tox21/transform.py:60]: I guess here we can remove the `is`?

**Review by MicPie — APPROVED** [review]
lgtm

**Pull request merged**